src/components/products/Classes/Product.ts

```diff
@@ -0,0 +1,14 @@
+interface Product {
```

Comment:
Don't use `Classes` as the directory name. There really aren't any classes as far as React functional components and we'd always call these `types`. It could create confusion if others were working on your app.